Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.py

```diff
@@ -0,0 +1,761 @@
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this HelloWorld Feed implementation, no special attributes defined
+    """
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+
+        result = []
+
+        res = self._http_request('GET',
+                                 url_suffix='',
+                                 full_url=self._base_url,
+                                 resp_type='text',
+                                 )
+
+        # In this case the feed output is in text format, so extracting the indicators from the response requires
+        # iterating over it's lines solely. Other feeds could be in other kinds of formats (CSV, MISP, etc.), or might
+        # require additional processing as well.
+        try:
+            indicators = res.split('\n')
+
+            for indicator in indicators:
+                # Infer the type of the indicator using 'auto_detect_indicator_type(indicator)' function
+                # (defined in CommonServerPython).
+                if auto_detect_indicator_type(indicator):
+                    result.append({
+                        'value': indicator,
+                        'type': auto_detect_indicator_type(indicator),
+                        'FeedURL': self._base_url
+                    })
+
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data as indicator. \n\nError massage: {err}')
+        return result
+
+    def get_hashes(self, params: Dict[str, str], limit: int) -> List:
+        res = self._http_request(
+            'GET',
+            url_suffix='',
+            full_url="http://api.cybercure.ai/feed/get_hash",
+            resp_type='text',
+        )
+
+        json_payload = json.loads(res)
+        result = []
+        # all_data = json_payload.get("data")
+        all_urls = json_payload.get("data").get("hash")
+        for data in all_urls:
+            result.append(
+                {
+                    'value': data,
+                    "type": "File",
+                    # "reputation":"Bad",
+                }
+            )
+        return result
+
+    """
+    """
+
+    def get_urls(self, limit: int) -> List:
+        res = self._http_request('GET',
+                                 url_suffix='',
+                                 full_url="http://api.cybercure.ai/feed/get_url",
+                                 resp_type='text',
+                                 )
+
+        json_payload = json.loads(res)
+        result = []
+        # all_data = json_payload.get("data")
+        all_urls = json_payload.get("data").get("urls")
```

Comment:
Also, could it be that response will not contain data entry?
```suggestion
        all_urls = res.get("data").get("urls")
```